open-api.yaml

```diff
@@ -0,0 +1,1614 @@
+          type: object
+          additionalProperties:
+            type: object
+            additionalProperties:
+              type: integer
+          description: |
+            [Distribution of the given facets](/reference/features/search_parameters.html#the-facets-distribution).
+          example:
+            genres:
+              action: 273
+              animation: 118
+              adventure: 132
+              fantasy: 67
+              comedy: 475
+              mystery: 70
+              thriller: 217
+        exhaustiveFacetsCount:
+          type: boolean
+          description: Whether `facetsDistribution` is exhaustive.
+        processingTimeMs:
+          type: integer
+          description: Processing time of the query.
+        query:
+          type: string
+          description: Query originating the response.
+          example: ninja
+      required:
+        - hits
+        - offset
+        - limit
+        - nbHits
+        - exhaustiveNbHits
+        - processingTimeMs
+        - query
+      title: ''
+    update:
+      type: object
+      description: |
+        MeiliSearch is an asynchronous API. It means that the API does not behave as you would typically expect when handling the request's responses.
+
+        Some operations are put in a queue and will be executed in turn (asynchronously). In this case, the server response contains the identifier to track the execution of the operation.
+
+        [Learn More](/learn/advanced/asynchronous_updates.html).
+      additionalProperties: false
+      properties:
+        status:
+          description: The state of the operation.
+          enum:
+            - enqueued
+            - processed
+            - failed
+        updateId:
+          type: integer
+          description: The id of the update.
+        type:
+          type: object
+          additionalProperties: false
+          required:
+            - name
+            - number
+          properties:
+            name:
+              type: string
+            number:
+              type: integer
+        enqueuedAt:
+          $ref: '#/components/schemas/timestamp'
+        duration:
+          type: number
+          description: The number of seconds taken to complete the operation.
+        processedAt:
+          $ref: '#/components/schemas/timestamp'
+        error:
+          type: string
+          description: |
+            A string describing [the error that occurred](/errors/).
+      required:
+        - status
+        - updateId
+        - type
+        - enqueuedAt
+    synonyms:
+      type: object
+      description: List of associated words treated similarly.
+      additionalProperties:
+        type: array
+        items:
+          type: string
+      example:
+        wolverine:
+          - xmen
+          - logan
+        logan:
+          - wolverine
+          - xmen
+        wow:
+          - world of warcraft
+    stopWords:
+      type: array
```

Comment:
can be null

Reply:
It can be an empty array but not null as a model! But it can be set with null value in POST methods